src/rest_api/classes/anatomy_term.clj

```diff
@@ -1,13 +1,16 @@
 (ns rest-api.classes.anatomy-term
   (:require
     [rest-api.classes.gene.widgets.external-links :as external-links]
-    ;[rest-api.classes.anatomy_term.widgets.overview :as overview]
-    [rest-api.classes.anatomy_term.widgets.ontology-browser :as ontology-browser]
+    ;[rest-api.classes.anatomy-term.widgets.overview :as overview]
+    [rest-api.classes.anatomy-term.widgets.references :as references]
+    [rest-api.classes.anatomy-term.widgets.ontology-browser :as ontology-browser]
     [rest-api.routing :as routing]))
 
 (routing/defroutes
-  {:entity-ns "anatomy-term"
+  {:entity-ns "anatomy-function"
```

Comment:
I do not. I think it would be interesting to find out about the schema to find out why it is the way it is. I wish I had deeper insight